app/pages/project/instances/actions.tsx

```diff
@@ -119,8 +115,8 @@ export const useMakeInstanceActions = (
         },
         {
           label: 'Resize',
-          onActivate() {
-            navigate(pb.instanceResize(instanceSelector))
+          onActivate: () => {
+            options.onResizeClick && options.onResizeClick(instance)
```

Comment:
Interestingly this does not fail when run locally.

But also there must be a better way of doing this? `onResizeClick` should not be optional but our declaration of options as `{}` insists on it.